nx-X11/lib/X11/XKBMisc.c

```diff
@@ -666,7 +666,7 @@ register int	i;
 int		width,nOldGroups,oldWidth,newTypes[XkbNumKbdGroups];
 
     if ((!xkb) || (!XkbKeycodeInRange(xkb,key)) || (!xkb->map) ||
-	(!xkb->map->types)||(!newTypes)||((groups&XkbAllGroupsMask)==0)||
```

Comment:
> It could be nice to add (at a later time) some spaces around the &s in the code, it's hard to read 
> and makes things look like a single identifier at a first glance. 

I am indeed thinking about running the format scripts that has been applied to X.org's Xserver way back. Only in coordination with the team, to not wrack some pending PRs / work-in-progress.